config/locales/pl/restroom.pl.yml

```diff
@@ -20,6 +20,9 @@ pl:
     guess_location: 'Obecna lokalizacja urządzenia'
     directions: 'Wskazówki'
     directions_hint: '(np.: na trzecim piętrze, obok baru, koło przebieralni...)'
+    distance:
+      miles: mile
+      kilometers: kilometrów
```

Comment:
Hi, @tusindfryd, @katafrakt.

It has been a while since you contributed the Polish translation for Refuge Restrooms (#475), so I'm sorry to bother you if you are busy.

In any case, we are now trying to add translations for the words "mile" and "kilometer", but we only have a machine translation from Google Translate so far. I am wondering if you could confirm that this translation looks good, or suggest the correct translation?

Thank you very much.

Sincerely,

\- DeeDeeG

Reply:
Hi,

In Polish numerals differ slightly: 1 kilometr, 2 - 4 kilometry, 5 - 21 kilometrów, 22 - 24 kilometry, 25 - 31 kilometrów, and so on. I think I would personally go with just writing `km` tbh.

With miles it would be: 1 mila, 2 - 4 mile, 5 - 21 mil, 22 - 24 mile, 25 - 31 mil, and so on. There is no abbreviation for miles in Polish, but no one uses them, so ¯\_(ツ)_/¯ 

So yeah. That's fun.

Reply:
Hmm.

Thank you for the information. I suppose we'll have to think of how we want to do this.

There are ways in Rails to use different pluralizations for different numbers. But it would be a bit trickier to set up. And I'm a little nervous at adding math and branching `if` - `then` logic in the results page for performance and complexity reasons. But it is possible.

Reply:
Wow, it's really interesting the way Polish handles numbers and plurals. Maybe we should go with the abbreviations ('km' and 'mi') anyway in this case.